README.md

```diff
@@ -167,5 +187,18 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+class Lamp < ActiveRecord::Base
+  has_many :genies
+end
```

Comment:
The above example is supposed to be a one to one relationship instead, so it would be `belongs_to :genie ` although I can see why you would think otherwise!